gatsby-config.js

```diff
@@ -63,5 +64,13 @@ module.exports = {
     },
     `gatsby-redirect-from`,
     `gatsby-plugin-meta-redirect`,
+    {
+      resolve: "gatsby-plugin-robots-txt",
+      options: {
+        host: "https://civicactions.com",
```

Comment:
I believe host should not include protocol: https://yoast.com/ultimate-guide-robots-txt/#host-directive

Reply:
Good catch... That came over in the example cut and paste from the plugin page.